pkg/agent/multicast/mcast_discovery.go

```diff
@@ -97,14 +99,49 @@ func (s *IGMPSnooper) queryIGMP(group net.IP, versions []uint8) error {
 		if err != nil {
 			return err
 		}
-		if err := s.ofClient.SendIGMPQueryPacketOut(igmpQueryDstMac, mcastAllHosts, openflow13.P_NORMAL, igmp); err != nil {
+		// outPort sets the output port of the packetOut message. We expect the message expected to go through OVS pipeline
```

Comment:
We expect the IGMP query packet to go through OVS pipeline by setting the output port number as 0. 

It is not necessary to say that "Openflow can help to this....". But if you want to mention the final value in the OpenFlow message, please think about this description:
"The final OpenFlow message will use a standard OpenFlow port number `OFPP_TABLE = 0xfffffff9` corrected by ofnet.